main.go

```diff
@@ -66,6 +68,10 @@ func (items *Items) ParseXML(filename string) error {
 		return fmt.Errorf("failed to close `%s`: %v", filename, err)
 	}
 
+	log.WithFields(log.Fields{
+		"function": "ParseXML()",
```

Comment:
When you have only one attribute to add just use log.WithField("function", "parseXML") instead of .WithFields{}